update.md

```diff
@@ -0,0 +1,29 @@
+# All local accounts requires verification. 
+**In order to let third party accounts to login with an email of "uniqueID@loopback.provider.com" example "123456@loopback.twitter.com" 
```

Comment:
Sure. 
I will create a task for that. For the mean time, I guess we can update the passport document in case this info could be useful for usage. I have seen some questions around that issue.